robot/basestation/static/js/pages/rover.js

```diff
@@ -595,7 +590,7 @@ function gameLoop () {
       if (steering > maxSoftSteering) {
         steering = maxSoftSteering
       }
-      // lastCmdSent = new Date().getTime()
+      // setTimeSinceCMD()
```

Comment:
why is this commented?

Reply:
I don't know the reasoning for why `lastCmdSent = new Date().getTime()` was commented out. So I just left it like that after replacing it with my function, in case there was a reason for it. Should I just uncomment it ?